packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,146 @@
+      data).
+    - Defines decimal precision (e.g., 2 places for currency, 3 for scientific
+      data) and coerces values to enforce it.
+    - Does not use increment/decrement buttons.
+    - Supports locale-specific formatting (e.g., 1.23 vs. 1,23).
+
+## Requirements
+
+1. Support locale-specific formatting for decimal values (e.g., 10212,456 in
+   German).
+2. Allow specifying decimal precision (e.g., 2 places → 123.45).
+3. Provide validation-only mode for max/min, supporting both coercion and
+   non-coercion on blur.
+
+### Anatomy
+
+![ ](spec-images/number-field-anatomy.png)
+
+| Element             | Description                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------ |
+| Label               | Name for the field.                                                                              |
+| Required            | Indicates whether the field is required or optional.                                             |
+| Placeholder         | Shows an example of the data required in the field or a description of the purpose of the field. |
+| Numeric value       | Current control value.                                                                           |
+| Increment/Decrement | Changes numeric value by the step size.                                                          |
+| Error status icon   | Appears when a user has entered invalid data in a field.                                         |
+| Error text          | Appears when a user has entered invalid data in a field.                                         |
+
+### Related Components
+
+- [Number Field](https://nimble.ni.dev/storybook/index.html?path=/docs/components-number-field--docs)
+
+## Design
+
+### Configuration
+
+Proposed attributes:
+
+| Attribute               | Description                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| coerce-invalid          | Boolean - Whether to coerce invalid max/min values on blur.                                                               |
+| maximum-fraction-digits | Maximum number of digits after the decimal separator.                                                                     |
+| minimum-fraction-digits | Minimum number of fraction digits. Values with fewer digits will be right-padded with zeros to meet the specified length. |
+
+### Behavior
+
+#### Mouse Interactions
+
+Hovering over the field input indicates interactivity.
+
+Clicking on the field input focuses the input and places a caret. If no value
+exists, the caret is placed at the far left. Otherwise, it is placed where
+clicked.
+
```

Comment:
Double click to select all? We get this for free, but might be worth specifying to ensure it doesn't break.